src/pages/index.js

```diff
@@ -127,19 +150,94 @@ function closeModal(modal) {
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
-  closeModal(editModal);
+  const submitButton = evt.submitter;
+  submitButton.textContent = "Saving...";
```

Comment:
You have `setButtonText` for button text changing. Please, use it everywhere not to duplicate the code